### PR TITLE
change default kubeconfig location

### PR DESCRIPTION
**Pull request by deads2k**
Changes the default kubeconfig location to `~/.kube/config`.  Significantly smaller than I expected.

@liggitt @smarterclayton

### CODE REVIEW

**Comment by deads2k** [comment]
@stefwalter fyi

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
These changes are LGTM - Jordan has final sign off (if he so chooses)

File: pkg/cmd/cli/config/loader.go [hunk 1]
```diff
@@ -14,19 +14,19 @@ import (
...
+	OpenShiftConfigPathEnvVar      = "KUBECONFIG"
 	OpenShiftConfigFlagName        = "config"
-	OpenShiftConfigHomeDir         = ".config/openshift"
+	OpenShiftConfigHomeDir         = ".kube"
 	OpenShiftConfigHomeFileName    = "config"
 	OpenShiftConfigHomeDirFileName = OpenShiftConfigHomeDir + "/" + OpenShiftConfigHomeFileName
 )
 
-var OldRecommendedHomeFile = path.Join(os.Getenv("HOME"), ".config/openshift/.config")
+var OldRecommendedHomeFile = path.Join(os.Getenv("HOME"), ".kube/.config")
```
Comment (liggitt) [inline pkg/cmd/cli/config/loader.go]: `.kube/.config` was never a path anyone used as far as I know, can we remove this? fine in a follow-up if you like

**Comment by liggitt** [comment]
other than removing OldRecommendedHomeFile for hygiene, LGTM

File: pkg/cmd/cli/config/loader.go (hunk 1 above)
Comment (deads2k) [inline pkg/cmd/cli/config/loader.go]: > .kube/.config was never a path anyone used as far as I know, can we remove this? fine in a follow-up if you like

Followup

**Comment by liggitt** [comment]
needs broadcasting to every mailing list in existence... I'll let @smarterclayton do the honors and wait for the mobs with pitchforks

**Comment by deads2k** [comment]
> needs broadcasting to every mailing list in existence... I'll let @smarterclayton do the honors and wait for the mobs with pitchforks

I'm in and out tomorrow, so I may get to miss some of the excitement :)

**Comment by smarterclayton** [comment]
Hold on to your butts

**Comment by liggitt** [comment]
[merge]

**Comment by derekwaynecarr** [comment]
MADNESS!

**Comment by derekwaynecarr** [comment]
(and now i know why my vm just stopped working)

**Comment by stefwalter** [comment]
You guys really doubled down on this ... without having auto migration. But thanks for putting in the work to make the change.

**Comment by kargakis** [comment]
> (and now i know why my vm just stopped working)

and-if-you-don-t-know-now-you-know

**Comment by stevekuznetsov** [comment]
can we keep a tally of all the hours people spent that eventually led them to this PR?

**Comment by thesteve0** [comment]
This was a very poor decision and communicated poorly. I am also quite sure you broke the class VM working with OC less than a week before my class.

**Comment by smarterclayton** [comment]
Sorry

**Comment by fabianofranz** [comment]
Can I add that I got kind of frustrated for knowing about this change and its motivations through the public list? If I had a vote I'd say to go back to the behavior we had in the beginning (try the openshift config location and fallback to kube location).

**Pull request merged**